packages/amplication-data-service-generator/src/server/resource/service/create-service.ts

```diff
@@ -392,27 +386,7 @@ function createTemplateMapping(
     UPDATE_ARGS: builders.identifier(`${entityType}UpdateArgs`),
     DELETE_ARGS: builders.identifier(`${entityType}DeleteArgs`),
     DELEGATE: delegateId,
-    CREATE_ARGS_MAPPING: createMutationDataMapping(
```

Comment:
do we still need the function "createMutationDataMapping"? where is it defined?

Reply:
deleted @yuval-hazaz